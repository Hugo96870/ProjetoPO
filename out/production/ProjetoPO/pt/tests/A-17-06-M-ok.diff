*** tests/A-17-06-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-17-06-M-ok.outhyp	2020-12-03 15:05:37.240548700 +0000
***************
*** 50,81 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do fornecedor: Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Gestão de Transacções
! 1 - Visualizar
! 2 - Registar venda
! 3 - Registar encomenda
! 4 - Pagar
! 0 - Sair
! Escolha uma opção: Identificador da transacção: 0|P2|2616|0
! 2H2|303
! 3O3|125
! 6M2|101
! Gestão de Transacções
! 1 - Visualizar
! 2 - Registar venda
! 3 - Registar encomenda
! 4 - Pagar
! 0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir
! 2 - Guardar
! 3 - Mostrar Data
! 4 - Avançar Data
! 5 - Gestão de Produtos
! 6 - Gestão de Clientes
! 7 - Gestão de Fornecedores
! 8 - Gestão de Transacções
! 9 - Menu de Consultas
! 10 - Ver Saldo Global
! 0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 50,53 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do fornecedor: Identificador do fornecedor: Identificador do produto: Quantidade: Quantidade: Quantidade: 
\ No newline at end of file
