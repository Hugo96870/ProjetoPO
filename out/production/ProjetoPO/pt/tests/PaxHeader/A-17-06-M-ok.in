29 mtime=1606841711.26390135
