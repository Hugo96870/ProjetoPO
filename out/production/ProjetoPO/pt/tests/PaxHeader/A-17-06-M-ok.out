30 mtime=1606841711.264605588
